corehq/apps/app_manager/models.py

```diff
@@ -823,8 +824,12 @@ def wrapped_xform(self):
     def validate_form(self):
         vc = self.validation_cache
         if vc is None:
+            # formtranslate requires all attributes to be valid xpaths, but
```

Comment:
This seems wrong to me. Do we really have no attribute values that are not valid xpath? Said another way, I'm surprised that every single attribute in our xforms contains a valid xpath expression.